docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -149,6 +149,9 @@ included to provide information about the search on each cluster.
     "total": 1,
```

Comment:
Let's add a note (`<1>`)  to this section stating something like:

> This section of counters shows all possible cluster search states and how many cluster searches are currently in that state.

This was a key motivation for why we want this change - so end users will know what all possible states are for parsing the sections below.

Reply:
And maybe combine that new note with the current `<2>` note and remove the `<2>` note as that would be redundant.

Reply:
Done!